content/docs/guides/crossguard/configuration.md

```diff
@@ -0,0 +1,132 @@
+
+Configuration schema is defined per policy, and then the actual configuration can be set via a form in the Pulumi Console or using a JSON file.
+
+Configuration is currently supported for the TypeScript and JavaScript Policy SDKs. Python support for configuration is [coming soon](https://github.com/pulumi/pulumi-policy/issues/210).
+
+## Writing Configurable Policy Packs
+
+### Enforcement Level
+
+The Policy SDK will automatically make the enforcement levels of all policies configurable. You can configure the enforcement level for all policies in a Policy Pack or configure it for each individual policy level. The top level enforcement level under `all` will override the existing enforcement levels of all policies in the Policy Pack. A configured enforcement level for a specific policy will override the policy's existing enforcement level as well as the top level configured enforcement level.
+
+In the example configuration below, all policies in the Policy Pack would be `disabled` with the exception of the `a-policy` policy, which would be `mandatory`.
+
+```json
+{
+    "all": {
+        "enforcementLevel": "disabled",
+    },
+    "a-policy": {
+         "enforcementLevel": "mandatory",
+    }
+}
+```
+
+### Custom Configuration
+
+Policy authors can denote the schema for a particular Policy's configuration using the `configSchema` field. The configuration schema uses [JSON Schema](https://json-schema.org/) to describe the acceptable configuration for a Policy.
+
+The below example shows a  `ResourceValidationPolicy` that has a required `message` field, which must be between five and fifty characters. Then, from within the `validateResource` function the configuration value can be accessed using `args.getConfig`.
+
+```typescript
+const examplePolicy: ResourceValidationPolicy = {
+    name: "example-policy-with-schema",
+    ...
+    configSchema: {
+        properties: {
+            message: {
+                type: "string",
+                minLength: 5,
+                maxLength: 50,
+                default: "This is a configurable message."
+            }
+        },
+        required: ["message"],
+    },
+    validateResource: validateTypedResource(aws.s3.Bucket, (_, args, reportViolation) => {
+        const config = args.getConfig<{ message?: string }>();
+        reportViolation("Here is the configurable message: " + config.message);
+    }),
+}
+```
+
+At the top level, the `configSchema` for each Policy has only the `properties` and `required` fields. Within the `properties` field, you may use the entire breadth of [validations keywords provided by the JSON Schema](https://json-schema.org/draft/2019-09/json-schema-validation.html#rfc.section.6). Validation keywords are fields like `minLength` and `maxLength` in the above example.
+
+## Configuring a Policy Pack
+
+You can configure a Policy Pack via the `pulumi` CLI or the Pulumi Console.
+
+### Using the Pulumi CLI
```

Comment:
I find the flow of these headings confusing:

- Using the Pulumi CLI
  - Running Policy Packs Locally
  - Validating Configuration
  - Enabling the Policy Pack
- Using the Pulumi Console

It starts off talking about running policy packs locally, but then gets into validating configuration and enabling the policy pack, which are about using policy packs with the Pulumi Console (via CLI commands).

I wonder if it's make more sense to organize this around the two ways of using Policy Packs:

- Running Policy Packs Locally
- Using the Pulumi Console

And then under "Using the Pulumi Console", first show how the UI itself can be used, but then also have sections about how the CLI can be used to validate and enable policy packs published to the service.